libs/platform/src/lib/components/form/input/input.component.html

```diff
@@ -12,7 +12,13 @@
     [attr.placeholder]="placeholder"
     [disabled]="disabled"
     [compact]="contentDensity == 'compact'"
-    [state]="status === 'error' ? 'invalid' : ''"
+    [state]="status"
+    [readOnly]="readonly"
     (blur)="_onFocusChanged(false)"
     (focus)="_onFocusChanged(true)"
+    [attr.aria-disabled]="disabled"
+    [attr.aria-readonly]="readonly"
+    [attr.aria-controls]="'input_ ' + 'id'"
```

Comment:
I don't think we need the `aria-controls` attribute, as the input element is not related to anything in this context. Normally `aria-controls` is used to establish a relationship, for example a "button" and a "dropdown menu".